### DISCUSSION TITLE
[HTML Sanitizer] option to avoid stripping links with relative scheme, e.g. "//example.com"

### DISCUSSION THREAD

**Discussion by virtualize** [symfony/symfony#58071] [Q&A] [answered]
HTML Sanitizer seems to strip attributes that contain links with a relative scheme, e.g. "//example.com". In my example a YouTube url. If I regex replace the `//` with `https://` HTML Sanitizer keeps the link.

Is there any option/config to keep these links with relative scheme by default? 

Thanks for any pointers

**Comment by stof** [comment]
By default, relative links are not allowed. I think this will work if you allow relative urls in the config.

Note however that there is no reason to use scheme-relative URLs. Even if your own website is loaded using HTTP, you should use a HTTPS link for Youtube (they would redirect you to HTTPS anyway if you use a HTTP URL)

**Comment by virtualize** [comment, marked as answer]
Hi Stof, thanks for your fast answer!

I do import this HTML from an API (Zendesk) so I have no control how they send urls.

Setting `allow_relative_links: true` does not work unfortunately. Should I open an issue?

**Reply by stof** [reply]
yes, please open an issue.

**Reply by virtualize** [reply]
done. https://github.com/symfony/symfony/issues/58073
